### DISCUSSION TITLE
How can I change root path with .htaccess?

### DISCUSSION THREAD

**Discussion by iroshiva** [wordplate/wordplate#278] [Q&A] [answered]
Hi,

I'm struggling to deploy wordplate with apache.

I tried to set the public folder as root in a .htaccess file.

RewriteEngine On
RewriteRule ^(.*)$ public/$1 [L]

I can access to the website, but the assets are not loaded...

Can anyone help me ?

Thx

**Comment by vinkla** [comment, marked as answer]
I would recommend to use a hosting provider where you can set the document root of your project such as Fortrabbit or Laravel Forge. Another option is to edit the Apache configuration instead of `.htaccess` file.

**Comment by fiskhandlarn** [comment]
@iroshiva You should probably take a look at WordPress guidelines re .htaccess. Your RewriteRule was missing index.php for starters.
https://wordpress.org/support/article/htaccess/
